### PR TITLE
feat: Added ddb table restore option to the complete example

### PR DESCRIPTION
**Pull request by patrickmukumbu**
# Description

This PR was created to add the missing attributes to showcase a complete module coverage

## Features/Fixes/Patches/Changes list:

Please check the [CHANGELOG.md](https://github.com/boldlink/terraform-aws-dynamodb/blob/feature/module-full-coverage/CHANGELOG.md#120---2023-10-05)

### PR Owners Checklist:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you checked there aren't open [Pull Requests](../../../pulls) of upstream or parallel code that will cause conflicts?
* [x] Have you updated the `CHANGELOG.md`?
* [x] Have you updated the `README.md`(s)?
* [x] OWNER: Does your submission pass?
    * [x] Pre-commit (attach logs/print screen to this PR)
    * [x] Checkov/terrascan (attach logs/print screen to this PR)
    * [x] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully run tests with your changes locally?
* [ ] First PR? Have you deleted `## How to use this template...` from the root `.README.md`?

### PR Reviewers Checklists?
* [ ] Are all your comments/changes resolved?
* [ ] Are you happy with the OWNER checklists and additional information provided?
* [ ] Does your review tests pass?
    * [ ] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,43 @@
...
+# AWS DynamoDB Terraform Module
 
-## Description
+This Terraform module provides an efficient and secure method to create an AWS DynamoDB table along with its item(s), encapsulating best practices and simplifying the deployment of scalable, serverless NoSQL databases.
 
-This terraform module creates a dynamodb table and item(s)
+## 🚀 Features
 
-Examples available [`here`](https://github.com/boldlink/terraform-aws-dynamodb/tree/main/examples)
+- **Effortless Deployment**: Streamline the creation of DynamoDB tables with predefined configurations and optional item population.
```
Comment (j0hnmwangi) [inline README.md:19]: Should we add these to be part of the Why Use This Module over starndard resources section ?

File: README.md [hunk 2]
```diff
@@ -11,13 +11,43 @@
...
+
+- **Encryption Support**: Ensures your data is secure by:
+  - Enabling encryption at rest using AWS KMS.
+  - Automatically creating a KMS key if encryption is specified and a custom key isn’t provided.
+
+- **Adherence to Security Best Practices**: This module:
+  - Follows AWS's best practices for secure and compliant infrastructure as code.
+  - Utilizes [Checkov](https://www.checkov.io/) to scan and mitigate potential vulnerabilities, ensuring your DynamoDB deployment is secure and compliant.
+
+## 🚀 Quick Start
```
Comment (j0hnmwangi) [inline README.md]: Hi Patrick, I think we may not need this section

**Review by j0hnmwangi — CHANGES_REQUESTED** [review]
Please see my comments

File: README.md (hunk 1 above)
Comment (patrickmukumbu) [reply README.md:19]: Yes, modified

File: README.md (hunk 2 above)
Comment (patrickmukumbu) [reply README.md]: You are right, I have removed it. Forgot to remove it earlier when I was pushing the changes

**Review by j0hnmwangi — APPROVED** [review]

**Pull request merged**